mergify_engine/tests/unit/test_command.py

```diff
@@ -84,3 +92,271 @@ def test_command_loader_wuth_defaults():
         "labels": [],
         "label_conflicts": "conflicts",
     }
+
+
+@pytest.mark.asyncio
+async def test_run_command_without_rerun_and_without_user(redis_cache):
+
+    client = mock.MagicMock()
+    client.auth.installation.__getitem__.return_value = 123
+    sub = subscription.Subscription(
```

Comment:
I think you can also merge the rerun test with those you have parametized. Or at least create a helper to build the context since they are always the same.